lib/src/matchers/jest_dom/has_form_values.dart

```diff
@@ -178,5 +176,5 @@ JsObject _getJsFormOrFieldSet(Element form) => JsObject.fromBrowserObject(form);
 /// Wraps the [HTMLFormElement.elements](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/elements) property.
 List<Element> getFormElements(Element form) {
   assert(form is FormElement || form is FieldSetElement);
-  return convertToArray<Element>(_getJsFormOrFieldSet(form)['elements'] as JsObject);
+  return convertToArray<Element>(_getJsFormOrFieldSet(form)['elements'] as JsObject?);
```

Comment:
If `_getJsFormOrFieldSet(form)['elements']` is null, I don't think `convertToArray` will return the expected type of `List<Element>`.